### PR TITLE
Updated helloBEEprusa printer profiles

### PR DESCRIPTION
**Pull request by marcosfg**
Upgraded helloBEEprusa profile to resolve some print quality and clogging issues.

### CODE REVIEW

**Comment by nallath** [comment]
For our reference; CURA-3916

File: resources/definitions/helloBEEprusa.def.json [hunk 1]
```diff
@@ -10,21 +10,28 @@
         "category": "Other",
         "platform": "BEEVERYCREATIVE-helloBEEprusa.stl",
         "platform_offset": [-226, -75, -196],
-        "file_formats": "text/x-gcode"
+        "file_formats": "text/x-gcode",
+		"machine_extruder_trains":
```
Comment (Ghostkeeper) [inline resources/definitions/helloBEEprusa.def.json]: You're mixing tabs with spaces here.

File: resources/definitions/helloBEEprusa.def.json [hunk 2]
```diff
@@ -10,21 +10,28 @@
...
+        {
+            "0": "hBp_extruder_left",
+            "1": "hBp_extruder_right"
+        }
     },
 
     "overrides": {
+	    "machine_extruder_count": { "default_value": 2 },
```
Comment (Ghostkeeper) [inline resources/definitions/helloBEEprusa.def.json]: And here too.

File: resources/definitions/helloBEEprusa.def.json [hunk 3]
```diff
@@ -10,21 +10,28 @@
...
         "machine_width": { "default_value": 185 },
         "machine_depth": { "default_value": 200 },
         "machine_height": { "default_value": 190 },
         "machine_heated_bed": { "default_value": true },
         "machine_center_is_zero": { "default_value": false },
-        "material_print_temperature": { "default_value": 220 },
+        "material_print_temperature": { "default_value": 200 },
         "material_bed_temperature": { "default_value": 60 },
         "material_diameter": { "default_value": 1.75 },
+		"line_width": { "default_value": 0.48 },
```
Comment (Ghostkeeper) [inline resources/definitions/helloBEEprusa.def.json]: More tabs.

Comment (Ghostkeeper) [inline resources/definitions/helloBEEprusa.def.json]: Are you sure that a line width of 0.48 with a nozzle size of 0.4 is going to work?

Well, your call I guess :)

Comment (marcosfg) [inline resources/definitions/helloBEEprusa.def.json]: Yes, we have been testing it with these parameters and its working better than with 0.4.

Thank you for fixings the tabs.

**Pull request merged**